README.rst

```diff
@@ -120,7 +120,8 @@ Creating an instance:
     >>> res1 = Ok('yay')
     >>> res2 = Err('nay')
 
-Checking whether a result is ``Ok`` or ``Err``. With ``isinstance`` you get type safe
+Checking whether a result is ``Ok`` or ``Err``. You can either use ``is_ok`` and ``is_err`` functions
+or ``isinstance``. This way you get type safe.
 access that can be checked with MyPy. The ``is_ok()`` or ``is_err()`` methods can be
 used if you don't need the type safety with MyPy:
```

Comment:
Note to self, perhaps make `methods` and `functions` bold to distinguish them since `is_ok` and `is_err` have the same names